docs/08-05-orchestrate-kyma-upgrade.md

```diff
@@ -1,6 +1,6 @@
 # Orchestrate Kyma upgrade
 
-This tutorial shows how to upgrade Kyma Runtime using Kyma Environment Broker.
+This tutorial shows how to upgrade SAP BTP, Kyma runtime using Kyma Environment Broker.
 
 ## Prerequisites
```

Comment:
Please change all instances of Kyma Environment Broker to KEB (except the first one, of course)